recipes/openscenegraph/all/conandata.yml

```diff
@@ -2,3 +2,13 @@ sources:
   3.6.5:
     sha256: aea196550f02974d6d09291c5d83b51ca6a03b3767e234a8c0e21322927d1e12
     url: https://github.com/openscenegraph/OpenSceneGraph/archive/OpenSceneGraph-3.6.5.tar.gz
+patches:
+  3.6.5:
+    - patch_file: patches/0001-Don-t-install-pdbs.patch
```

Comment:
Don't use patches for this. This makes adding new versions harder. Remove pdbs files in `package()`